README.md

```diff
@@ -81,3 +81,21 @@ npm install
 ```
 npm start
 ```
+
+#### Set up ESLint & Prettier
```

Comment:
You forgot to update the eslint and prettier section in here(`README.md`).

- Make it match the eslint part in `CONTRIBUTING.md` or just add a link to that part of the setup in the `CONTRIBUTING.md` file
- Also I don't think the part about installing the packages any useful because all the packages will only work when you `npm install` which I'm pretty sure they would have done setting up the frontend and backend. I think the most important part of this is the recommendation to install the respective extension since they do help although not critical since running `npm run lint` takes care of the styling